sdks/python/apache_beam/io/aws/s3filesystem.py

```diff
@@ -281,6 +281,25 @@ def checksum(self, path):
     except Exception as e:  # pylint: disable=broad-except
       raise BeamIOError("Checksum operation failed", {path: e})
 
+  def metadata(self, path):
+    """Fetch metadata fields of a file on the FileSystem.
+
+    Args:
+      path: string path of a file.
+
+    Returns:
+      :class:`~apache_beam.io.filesystem.FileMetadata`.
+
+    Raises:
+      ``BeamIOError``: if path isn't a file or doesn't exist.
+    """
+    try:
+      file_metadata = s3io.S3IO(options=self._options)._vars(path)
```

Comment:
hm in fact - I feel weird but shouldn't this be _status rathern than _vars - I cant find the _vars method...?

Reply:
Thanks for pointing out! It was indeed a mistake, a leftover of a renaming request.